app/helpers/redirect_destination_helper.rb

```diff
@@ -1,7 +1,7 @@
 module RedirectDestinationHelper
   def redirect_destination_select_options
     guide_select_options = Guide
-      .with_published_editions
+      .live
```

Comment:
Mainly out of curiosity… we have leading periods in this method and trailing in the guide_helper method. Do we have a standard way of doing this, and should we be following it?